packages/client/src/Compose/RecipientsInput.tsx

```diff
@@ -67,46 +101,130 @@ function reducer(state: State, action: Action): State {
   }
 }
 
-export default function RecipientsInput({ onRecipients, ...rest }: Props) {
+export default function RecipientsInput({
+  onRecipients,
+  fullWidth,
+  className,
+  ...rest
+}: Props) {
   const classes = useStyles()
   const [{ recipients, inputValue }, dispatch] = React.useReducer(reducer, {
     recipients: [],
     inputValue: ""
   })
+
   const [focused, setFocused] = React.useState(false)
 
+  const { data } = graphql.useGetMatchingAddressesQuery({
+    variables: { inputValue }
+  })
+
+  let suggestions = data ? data.addresses : []
```

Comment:
I would still like you to change this declaration to `const suggestions` because this variable should not be reassigned.